notebooks/week_1/01_preprocessing.py

```
@@ -0,0 +1,86 @@
+# Databricks notebook source
+# MAGIC %pip install ../hotel_reservations-2.2.0-py3-none-any.whl
+
+# COMMAND ----------
+dbutils.library.restartPython()
```

Ruff

Undefined name `dbutils`

(F821)